flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/data/AbstractTestFlinkAvroReaderWriter.java

```diff
@@ -80,20 +77,18 @@ private void writeAndValidate(Schema schema, List<Record> expectedRecords, int n
 
     // Write the expected records into AVRO file, then read them into RowData and assert with the
     // expected Record list.
-    try (FileAppender<Record> writer =
+    try (FileAppender<org.apache.iceberg.data.Record> writer =
```

Comment:
Do we really need the fully qualified name here? I'm reviewing on my phone, so I might miss something, but I was not able to find the reason

Reply:
No, we can use the short name. I will update.